src/constructs/autoscaling/asg.ts

```diff
@@ -164,8 +166,56 @@ export class GuAutoScalingGroup extends GuAppAwareConstruct(AutoScalingGroup) {
     // { UpdatePolicy: { autoScalingScheduledAction: { IgnoreUnmodifiedGroupSizeProperties: true }}
     if (!updatePolicy) {
       cfnAsg.addDeletionOverride("UpdatePolicy");
+      Tags.of(this).add("gu:rotated-by", "riff-raff");
+    } else {
+      Tags.of(this).add("gu:rotated-by", "cloudformation");
     }
 
     Tags.of(launchTemplate).add("App", app);
   }
+
+  signalOnHealthyTargetGroup(targetGroup: ApplicationTargetGroup): void {
```

Comment:
Worth a doc string? I imagine we'd want to indicate this function isn't intended for use by consumers.

Reply:
I think this would be fine for consumers to use if they're not using our EC2 patterns. Doc string is needed though!